extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -336,8 +353,56 @@ export class ConsentUI {
     classList.remove(consentUiClasses.in);
     this.isIframeVisible_ = false;
     removeElement(dev().assertElement(this.ui_));
+
+    // TODO (torch2424): Hide mask if publisher provides the option
+    // this.hideMaskElement_();
+  }
+
+  /**
+   * Disables scrolling on the document
+   * @private
+   */
+  disableScroll_() {
+    if (!this.scrollDisabled_) {
```

Comment:
Is `this.scrollDisabled_` set to true somewhere? I couldn't find it

Reply:
🙃 Yeah totally need to fix that